### ISSUE TITLE
Bootstrap version

### ISSUE THREAD

**Issue by thet** [plone/plonetheme.barceloneta#22]
It looks like, plonetheme.barceloneta took the Bootstrap framework as a basis and modified it directly instead of depending on it and overloading rules as needed.
I'm not sure if this is good or bad, maybe it's excellent. But I have the feeling, that this approach removes some flexibility, as it locks us to the version used when barceloneta was created.

**Comment by vangheem** [comment]
It took parts of it. I don't think(I could be wrong), it re-uses any of the class names, which is important. 

barceloneta doesn't *use* any version of bootstrap. It just borrowed some styles and techniques. The author didn't like the idea of using bootstrap directly. Having to deal with upgrading bootstrap versions, I think, was an advantage, not a disadvantage. I could be wrong though.

I'm fine with this approach....

**Comment by bloodbare** [comment]
We discussed a lot about this matter when we were doing the barceloneta theme. So we ended doing our own structured less infraestructure that does not depend on bootstrap and provides a compatibility layer. Bootstrap is not only a matter of css is also js, and in our case we are using patterns so it does not make sense to use only half of bootstrap.

If we wanna upgrade barceloneta less files to be compatible with a new bootstrap you just need to check the correspondent less file on barceloneta theme folder and change it to the one you want to upgrade (that provides the option to change the grid system to use any other you want just overwriting the gris.plone.less)

I know this aproach has its cons ( its not as simple to put a bootstrap theme on top of barceloneta theme ) but its not a big work to adapt it. We are used to use comercial bootstrap themes on the markup.

As my mine pros are depending on bootstrap css part is not really usefull, that ties our hands, creates a three layer system ( bootstrap - barceloneta - own theme ) that may end with lots of conflics. I love that we don't depend on bootstrap but we have the same compatibility. At the end is good for integrators that they know that the main css is not going to change as often as bootstrap is doing. Finally it does not make sense that we use modal/dropdown... css from bootstrap ( most of the css on bootstrap is realeted to js components ) as we have patterns...
